custom-plugin-rules/default-enum-match/default-enum-plugin.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function myRulesPlugin() {
+  return {
+    id: 'default-enum',
+    rules: {
+      oas3: {
```

Comment:
I think it could also apply to asyncapi too though?

Reply:
I'm unfamiliar with the asyncapi spec, but I'll add the key if you like.

Reply:
Done